breathecode/events/serializers.py

```diff
@@ -322,6 +358,8 @@ class Meta:
         exclude = ('visibility_settings', )
 
     def validate(self, data: dict[str, Any]):
+        if 'icon_url' not in data:
```

Comment:
This validation is doing thought the model, add a test when you is adding a event type and it fail and remove this conditional